tests/dirty_messages_1.json

```diff
@@ -0,0 +1,713 @@
+[
```

Comment:
This file is an artifact of the test and should be added to the .gitignore instead of committed here.